ui/app/AppLayouts/Wallet/panels/WalletFooter.qml

```diff
@@ -52,7 +54,10 @@ Rectangle {
             icon.name: "send"
             text: root.isCommunityOwnershipTransfer ? qsTr("Send Owner token to transfer %1 Community ownership").arg(root.communityName) : qsTr("Send")
             interactive: !d.isCollectibleSoulbound && networkConnectionStore.sendBuyBridgeEnabled
-            onClicked: root.launchSendModal()
+            onClicked: {
+                root.transactionStore.setSenderAccount(root.walletStore.selectedAddress)
```

Comment:
@Cuteivist A late question on this. Why do we need this preparation before opening the send modal?

Reply:
Thanks! 🍻 I'm looking into all places where send modal can be opened to align it with the design and I wasn't sure about this extra step!